src/screens/TopicPageScreen/elements/NavHeader.js

```diff
@@ -4,6 +4,7 @@ import {useNavigation} from '@react-navigation/core';
 import PropTypes from 'prop-types';
 
 import {useSafeAreaInsets} from 'react-native-safe-area-context';
+import FastImage from 'react-native-fast-image';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `FastImage` from `react-native-fast-image` is a good change as it can improve the performance of image loading and caching.



<!-- This is an auto-generated comment by OSS CodeRabbit -->